app/views/capture/index2.html.erb

```diff
@@ -1,4 +1,5 @@
-<script src="https://cdn.temasys.io/adapterjs/0.15.x/adapter.min.js"></script>
+<html lang="en" role="main">
+<%= javascript_include_tag "adapterjs/publish/adapter.min.js" %> 
```

Comment:
Ah, i missed this. I think we need to add this to package.json.

Reply:
https://www.npmjs.com/package/adapterjs

Reply:
Oh my bad, I think I missed that 😅

Reply:
Will need to update yarn.lock too...that way the yarn.lock file gets referenced in actions...

Reply:
Whew, this published to the live site. Getting a fix in now!

OK: https://github.com/publiclab/spectral-workbench/commit/1091951a46bf2a5f9ee9abc1b3e394936bdf976a

Let's test this out in stable or at least GitPod in the future! But seems all good now... will deploy shortly. Thanks!

Reply:
Oh I made another one at #704 😅